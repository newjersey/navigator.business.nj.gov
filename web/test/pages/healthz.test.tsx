import HealthCheck from "@/pages/healthz";
import { render } from "@testing-library/react";
import React from "react";

describe("HealthzPage", () => {
  it("renders correctly", () => {
    const subject = render(<HealthCheck />).baseElement;
    expect(subject).toMatchSnapshot();
  });
});
