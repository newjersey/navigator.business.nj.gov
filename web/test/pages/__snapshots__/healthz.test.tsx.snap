// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HealthzPage renders correctly 1`] = `
<body>
  <div>
    Application is healthy
  </div>
</body>
`;
